packages/protocol/src/action/protocols.ts

```diff
@@ -52,6 +53,14 @@ export interface NodeProtocol extends CollectionProtocol<EndpointProtocol> {
      */
     eventHandler: OccurrenceManager;
 
+    /**
+     * Event when data on the node changes.
+     */
+    stateChanged: Observable<[endpointId: EndpointNumber, clusterId: ClusterId, changes: AttributeId[]], MaybePromise>;
```

Comment:
Crap, this breaks quieter events.

I think this is still a good optimization but we should probably have the ProtocolService identify Q events and split the logic... For Q, filter from the datasource and add a listener to the online observable.

On the plus side, good reason to room LVL and CC tests even if cluster didn't change... Ws could just break cc up until maybe three separate groups so they can run in parallel.